resources/views/platform/components/game/heading.blade.php

```diff
@@ -1,19 +1,34 @@
 @props([
-    'consoleName' => 'Unknown Console',
+    'gameId' => 0,
     'gameTitle' => 'Unknown Game',
-    'iconUrl' => asset("assets/images/system/unknown.png")
+    'consoleId' => 0,
+    'consoleName' => 'Unknown Console',
+    'iconUrl' => asset("assets/images/system/unknown.png"),
+    'user' => null,
+    'userPermissions' => null,
 ])
 
 <?php
+use App\Community\Enums\UserGameListType;
+use App\Platform\Models\System;
+use App\Site\Enums\Permissions;
+
 // TODO: Migrate renderGameTitle to a Blade component.
 $renderedTitle = renderGameTitle($gameTitle);
+$type = UserGameListType::Play;
```

Comment:
Attempting to put this directly in the blade below
```
<x-game.add-to-list :gameId="$gameId" :type="{{ UserGameListType::Play }}" :user="$user" />
```

generated some weird error about an "unexpected >"

Reply:
I hit this issue fairly frequently. I believe the Blade renderer isn't able to interpolate static values. My workaround:

```
<?php
$userGameListTypePlay = UserGameListType::Play;
?>

:type="$userGameListTypePlay"
```